app/javascript/components/types.ts

```diff
@@ -795,3 +795,42 @@ export type Modifier =
   | 'placeholder-shown'
   | 'autofill'
   | 'read-only'
+
+export type CommunityVideoAuthorLinks = {
+  profile?: string
+  channel_url?: string
```

Comment:
An underscore in the property name?

Reply:
removed as it wasn't used. probably I copied it from the return object to transform it into a type, and left it like that. Thanks for spotting!